src/main/java/com/formstack/qa/pages/ViewLiveWebBrowserPage.java

```diff
@@ -99,6 +101,12 @@ public void fillOutNumber(String number, int numberFiledIndex) {
         numberInput.sendKeys(number);
     }
 
+    public void fillOutCreditCard(String ccNumber, int ccFiledIndex) {
```

Comment:
Looks like typo - I think it should be **ccFiledIndex**
```suggestion
    public void fillOutCreditCard(String ccNumber, int ccFieldIndex) {
```